tests/mocks.py

```diff
@@ -104,17 +110,17 @@ def off(self, *args, **kwargs):
     def connected(self):
         return self._mock_connected
 
-    def connect(self, *args, **kwargs):
+    async def connect(self, *args, **kwargs):
         self._mock_server.connection = self
         self._mock_connected = True
 
-    def disconnect(self, *args, **kwargs):
+    async def disconnect(self, *args, **kwargs):
         self._mock_server.connection = None
         self._mock_connected = False
 
 
 class MockEventLoop:
```

Comment:
We should dispose of this class, pydle should not be rolling its own event loop in this era.